packages/@stimulus/core/src/class_properties.ts

```diff
@@ -27,7 +25,13 @@ function propertiesForClassDefinition(key: string) {
       }
     },
 
-    [`has${capitalize(name)}`]: {
+    [`${key}Classes`]: {
+      get(this: Controller) {
+        return this.classes.get(key)?.split(" ") || []
```

Comment:
I'm new to typescript so if there is a better way to "annotate" this, please let me know :)